example-dvc-experiments/code/src/train.py

```diff
@@ -16,6 +18,93 @@
 BATCH_SIZE = 128
 
 
+def label_from_path(filepath):
+    """extracts "test", and 3 from a path like "images/test/3/00177.png" """
+    elements = filepath.split('/')
+    return (elements[1], int(elements[2]))
+
+def read_dataset(dataset_path):
```

Comment:
let's move both methods to utils? to keep is all about train, not these technical details on how do we read it ... it's too complicated for the get started I think ... wdyt?

Reply:
moved all longer functions to `util.py`